src/test/java/hw02/FamilyDeleteChildWithIndexTest.java

```diff
@@ -0,0 +1,45 @@
+package hw02;
+
+import hw01.Human;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class FamilyDeleteChildWithIndexTest {
```

Comment:
create one FamilyTest class and then write all your test inside this class, by giving them proper naming